src/types/actions.js

```diff
@@ -137,66 +169,66 @@ type ReceiveProfileAction =
     |}
   | {| +type: 'RECEIVE_ZIP_FILE', +zip: JSZip |}
   | {| +type: 'PROCESS_PROFILE_FROM_ZIP_FILE', +pathInZipFile: string |}
-  | {| +type: 'FAILED_TO_PROCESS_PROFILE_FROM_ZIP_FILE', error: any |}
+  | {| +type: 'FAILED_TO_PROCESS_PROFILE_FROM_ZIP_FILE', +error: any |}
```

Comment:
Why do we have `+error` here? :)

Reply:
It looks like I dispatched the error with the action, but then never ended up actually consuming the value. There error is `console.error` earlier, but otherwise not surfaced.

For the `+` change, it looks like I forgot to add it here. The `+` value in Flow typing indicates that an object will always be read only, and not changeable. This gives us immutability with no runtime costs. Action objects should never be mutated, so it is nice to reflect that in their type definition to enforce it.

https://flow.org/en/docs/types/interfaces/#toc-interface-property-variance-read-only-and-write-only